boards/beagle/beagleplay_cc1352/doc/index.rst

```diff
@@ -0,0 +1,192 @@
+| DIO11 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO12 | CC1352_RX    | AM62 UART6_TXD                      |
++-------+--------------+-------------------------------------+
+| DIO13 | CC1352_TX    | AM62 UART6_RXD                      |
++-------+--------------+-------------------------------------+
+| DIO14 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO15 | CC1352_BOOT  | AM62 GPIO0_13                       |
++-------+--------------+-------------------------------------+
+| DIO16 | CC1352_TDO   | TAG-CONNECT TDO                     |
++-------+--------------+-------------------------------------+
+| DIO17 | CC1352_TDI   | TAG-CONNECT TDI                     |
++-------+--------------+-------------------------------------+
+| DIO18 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO19 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO20 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO21 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO22 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO23 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO24 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO25 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO26 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO27 | LED1         | CC1352_LED1 yellow LED9             |
++-------+--------------+-------------------------------------+
+| DIO28 | LED2         | CC1352_LED2 yellow LED8             |
++-------+--------------+-------------------------------------+
+| DIO29 | RF_PA        | SubG/PA Antenna mux PA enable       |
++-------+--------------+-------------------------------------+
+| DIO30 | RF_SUB1G     | SubG/PA Antenna mux SubG enable     |
++-------+--------------+-------------------------------------+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+To flash, disable the existing driver that ties up the serial port and use
+the customized BSL Python script.
+
+1. Ensure the bcfserial or gb-beagleplay driver isn’t blocking the serial port. This can be done by
+   loading `/overlays/k3-am625-beagleplay-bcfserial-no-firmware.dtbo` or selecting uboot entry which
+   disables bcfserial/gb-beagleplay.
```

Comment:
```suggestion
1. Ensure the bcfserial or gb-beagleplay driver isn’t blocking the serial port. This can be done by
   loading :file:`/overlays/k3-am625-beagleplay-bcfserial-no-firmware.dtbo` or selecting uboot entry
   which disables bcfserial/gb-beagleplay.
```